tests/test_graphical_units/img_svg_resources/aabbb.svg

```diff
@@ -0,0 +1,27 @@
+</symbol>
+<symbol overflow="visible" id="glyph0-1">
+<path style="stroke:none;" d="M 5.296875 -2.171875 L 5.296875 -3.640625 L 3.765625 -3.640625 C 3.171875 -3.640625 2.726562 -3.507812 2.4375 -3.25 C 2.144531 -3 2 -2.609375 2 -2.078125 C 2 -1.597656 2.144531 -1.210938 2.4375 -0.921875 C 2.738281 -0.640625 3.140625 -0.5 3.640625 -0.5 C 4.140625 -0.5 4.539062 -0.65625 4.84375 -0.96875 C 5.144531 -1.28125 5.296875 -1.679688 5.296875 -2.171875 Z M 6.5 -4.328125 L 6.5 -0.6875 L 7.5625 -0.6875 L 7.5625 0 L 5.296875 0 L 5.296875 -0.75 C 5.035156 -0.425781 4.726562 -0.1875 4.375 -0.03125 C 4.03125 0.113281 3.628906 0.1875 3.171875 0.1875 C 2.398438 0.1875 1.785156 -0.015625 1.328125 -0.421875 C 0.878906 -0.828125 0.65625 -1.378906 0.65625 -2.078125 C 0.65625 -2.796875 0.914062 -3.347656 1.4375 -3.734375 C 1.957031 -4.128906 2.6875 -4.328125 3.625 -4.328125 L 5.296875 -4.328125 L 5.296875 -4.8125 C 5.296875 -5.332031 5.132812 -5.738281 4.8125 -6.03125 C 4.5 -6.320312 4.054688 -6.46875 3.484375 -6.46875 C 3.003906 -6.46875 2.625 -6.359375 2.34375 -6.140625 C 2.0625 -5.921875 1.882812 -5.597656 1.8125 -5.171875 L 1.203125 -5.171875 L 1.203125 -6.578125 C 1.617188 -6.753906 2.019531 -6.882812 2.40625 -6.96875 C 2.800781 -7.0625 3.1875 -7.109375 3.5625 -7.109375 C 4.519531 -7.109375 5.25 -6.867188 5.75 -6.390625 C 6.25 -5.921875 6.5 -5.234375 6.5 -4.328125 Z "/>
+</symbol>
+<symbol overflow="visible" id="glyph0-2">
+<path style="stroke:none;" d="M 1.53125 -0.6875 L 1.53125 -9.4375 L 0.390625 -9.4375 L 0.390625 -10.125 L 2.734375 -10.125 L 2.734375 -5.84375 C 2.960938 -6.269531 3.257812 -6.585938 3.625 -6.796875 C 3.988281 -7.003906 4.4375 -7.109375 4.96875 -7.109375 C 5.820312 -7.109375 6.515625 -6.769531 7.046875 -6.09375 C 7.585938 -5.425781 7.859375 -4.550781 7.859375 -3.46875 C 7.859375 -2.375 7.585938 -1.488281 7.046875 -0.8125 C 6.515625 -0.144531 5.820312 0.1875 4.96875 0.1875 C 4.4375 0.1875 3.988281 0.0859375 3.625 -0.109375 C 3.257812 -0.316406 2.960938 -0.640625 2.734375 -1.078125 L 2.734375 0 L 0.390625 0 L 0.390625 -0.6875 Z M 2.734375 -3.125 C 2.734375 -2.289062 2.890625 -1.65625 3.203125 -1.21875 C 3.523438 -0.78125 3.988281 -0.5625 4.59375 -0.5625 C 5.207031 -0.5625 5.671875 -0.804688 5.984375 -1.296875 C 6.304688 -1.785156 6.46875 -2.507812 6.46875 -3.46875 C 6.46875 -4.425781 6.304688 -5.144531 5.984375 -5.625 C 5.671875 -6.113281 5.207031 -6.359375 4.59375 -6.359375 C 3.988281 -6.359375 3.523438 -6.140625 3.203125 -5.703125 C 2.890625 -5.265625 2.734375 -4.628906 2.734375 -3.796875 Z "/>
+</symbol>
+</g>
+</defs>
+<g id="surface1">
+<g style="fill:rgb(100%,100%,0%);fill-opacity:1;">
+  <use xlink:href="#glyph0-1" x="30" y="32.375977"/>
+  <use xlink:href="#glyph0-1" x="38" y="32.375977"/>
+</g>
+<g style="fill:rgb(0%,50.196078%,0%);fill-opacity:1;">
+  <use xlink:href="#glyph0-2" x="46" y="32.375977"/>
```

Comment:
```suggestion
  <use xlink:href="#glyph0-2" x="46" y="32.375977" style="fill:rgb(50%,0%,0%)"/>
  ```

Reply:
I've made the changes by making sure the nearer style overrides the farther style, but also that style defined in `use` takes precedence over styles defined locally.